packages/validator/test/cli-validator/tests/threshold-validator.test.js

```diff
@@ -44,9 +44,9 @@ describe('test the .thresholdrc limits', function() {
       './test/cli-validator/mock-files/clean.yml'
     ]);
     // limit values invalid, so default limit, Number.MAX_VALUE, used
-    expect(exitCode).toEqual(0);
-
     const capturedText = getCapturedText(consoleSpy.mock.calls);
+    // originalError(`Captured text: ${JSON.stringify(capturedText, null, 2)}`);
```

Comment:
Should this be here? :)

Reply:
You'll see comments like this in several places among the testcases for the validator package.  It's helpful during debugging a test failure that's caused by changes to the validator's output in a specific situation.